components/event-publisher-proxy/pkg/handler/handlertest/handlertest.go

```diff
@@ -0,0 +1,329 @@
+package handlertest
```

Comment:
How about using `handler_test.go` as the name of the file?

Reply:
This file contains utils for handler testing. Same pattern is used in golang itself, check here: https://github.com/golang/go/blob/master/src/net/http/httptest/httptest.go.

Reply:
A comment of this [sort](https://github.com/golang/go/blob/master/src/net/http/httptest/httptest.go#L5)would make sense then.

Reply:
Done